tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "skipLibCheck": true,
     "resolveJsonModule": true,
     "useUnknownInCatchVariables": true,
-    "lib": ["ES2022", "DOM"],
```

Comment:
This was confusing typescript as it couldn't differentiate between the DOM Request and the cloudflare Request (with extras) and was giving us the DOM one instead, so the cf properties and other tricks were not available.